component/task.go

```diff
@@ -0,0 +1,7 @@
+package component
+
+type TaskLaunchInfo struct {
+	OciUrl               string
```

Comment:
I think a bit more details on each of these args would help. Some questions I have as a potential integrator right now: Is this always a "full" URL with a registry or can this be a shorthand like `nginx:tag` or can this even be shorter hand like `nginx` (implied latest), etc. 

No need to answer in this GH comment, would just like to see this in doc comments.